server/scripting/script_server.cpp

```diff
@@ -275,6 +280,29 @@ static void script_server_code_save(struct section_file *file)
   }
 }
 
+/**
+   Dump the bytecode of function passed and send to client to execute.
```

Comment:
Lua bytecode isn't portable between LuaJIT and PUC Lua and probably also between Lua versions, so this would impose tight constraints on the versions used. I would prefer using the source instead.